docs/pages/debugging/tools.mdx

```diff
@@ -191,6 +191,13 @@ To start debugging:
 
 This will attach VS Code to your running app.
 
+Alternatively, if you want a fully-featured IDE setup in VS Code, you might want to check out [Radon IDE](https://ide.swmansion.com/) (paid). This extension turns your editor into a powerful environment designed specifically for React Native and Expo projects, with advanced debugging, a network inspector, router integration, and other built-in tools. There's a 30-day free trial, so you can explore everything it offers before committing.
+
+<ContentSpotlight
+  alt="Debug code thanks to Radon IDE."
```

Comment:
```suggestion
  alt="Debugging code using Radon IDE."
```

It sounds more like a marketing tagline and a bit opinionated than a description for the image tag. Let's update to describe what's the actual image shows.

Reply:
Makes sense 👍